scripts/travis/Makefile

```diff
@@ -23,17 +26,29 @@ define DOCKER_JSON
 endef
 
 export DOCKER_JSON
-$(TARGETS):
-	echo "$$DOCKER_JSON" > /etc/docker/daemon.json
-	systemctl restart docker
+
+ifeq ($(UNAME),x86_64)
```

Comment:
Need a comment here to explain why we need different options

Reply:
done